go/vumitools/tasks.py

```diff
@@ -0,0 +1,53 @@
+import json
+from celery.task import task
+
+from go.base.command_utils import get_user_by_account_key
+from go.base.utils import vumi_api, vumi_api_for_user
+from go.config import get_conversation_definition
+from go.vumitools.metrics import get_conversation_metric_prefix
+
+
+@task(ignore_result=True)
+def send_recent_conversation_metrics():
+    conversation_details = get_and_reset_recent_conversations()
```

Comment:
We probably want to restructure this method to just get one VumiApi at the start, and then close it at the end:
```python
api = vumi_api()
conversation_details = get_and_reset_recent_conversations(api)
for ...:
     ...
     user_api = api.get_user_api(details["account_key"])
     conv = user_api.get_conversation(details["conv_key"])
     ...
api.close()
```
Maybe with a `try: ... finally: ...` block to ensure the `api.close()` call happens.